akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst

```diff
@@ -315,3 +292,97 @@ actor external actor of how to allocate shards or rebalance shards.
 
 For the synchronous case you can return the result via ``scala.concurrent.Future.successful`` in Scala or 
 ``akka.dispatch.Futures.successful`` in Java.
+
+################
+Akka Persistence
+################
```

Comment:
isn't this wrong, this is the same level as in the top.

I think we can have:

    Akka Persistence
    =============

and

    Persistence Plugin API
    ==================